tests/ttnn/unit_tests/operations/test_moreh_softmin.py

```diff
@@ -325,11 +325,13 @@ def test_softmin_backward_for_dim_nc(shape_dim, compute_kernel_options, device):
 
 @pytest.mark.parametrize(
     "shape_dim",
-    (((32, 32), 1),),  # single tile
+    [
+        [[32, 32], 1],
+    ],  # single tile
 )
 @pytest.mark.parametrize(
     "optional_output_tensor",
-    (True, False),
+    [True, False],
 )
 def test_softmin_optional_output_tensor(shape_dim, optional_output_tensor, device):
     device.enable_program_cache()
```

Comment:
should use `use use_program_cache` param

Reply:
this isn't a callback test so it doesn't really matter if we use program_cache or not right?